src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsModel.ts

```diff
@@ -421,6 +421,9 @@ export class InlineCompletionsModel extends Disposable {
 		const partialGhostTextVal = ghostTextVal.substring(0, acceptUntilIndexExclusive);
 
 		const positions = this._positions.get();
+		if (positions.length === 0) {
```

Comment:
I don't think this is necessary, `positions` should never have empty length here.